json/schema/user_import.schema.json

```diff
@@ -123,7 +137,8 @@
               }
             ]
           }
-        }
+        },
+        "additionalProperties": false
```

Comment:
Why that got added? :) Maybe it will be easier to just do `git revert` on the commit that @matepager mentioned.

Reply:
that got added in git revert, it was a later change, fixed it